subprojects/launcher/src/test/groovy/org/gradle/launcher/cli/DefaultCommandLineActionFactoryTest.groovy

```diff
@@ -219,34 +211,84 @@ class DefaultCommandLineActionFactoryTest extends Specification {
     def "displays version message"() {
         def version = DefaultGradleVersion.current()
         def expectedText = [
-            "",
-            "------------------------------------------------------------",
-            "Gradle ${version.version}",
-            "------------------------------------------------------------",
-            "",
-            "Build time:   $version.buildTimestamp",
-            "Revision:     $version.gitRevision",
-            "",
-            "Kotlin:       ${KotlinDslVersion.current().kotlinVersion}",
-            "Groovy:       $GroovySystem.version",
-            "Ant:          $Main.antVersion",
-            "JVM:          ${Jvm.current()}",
-            "OS:           ${OperatingSystem.current()}",
-            ""
+                "",
+                "------------------------------------------------------------",
+                "Gradle ${version.version}",
+                "------------------------------------------------------------",
+                "",
+                "Build time:   $version.buildTimestamp",
+                "Revision:     $version.gitRevision",
+                "",
+                "Kotlin:       ${KotlinDslVersion.current().kotlinVersion}",
+                "Groovy:       $GroovySystem.version",
+                "Ant:          $Main.antVersion",
+                "JVM:          ${Jvm.current()}",
+                "OS:           ${OperatingSystem.current()}",
+                ""
```

Comment:
Could this not just be a multiline string?

```
"""------------------------------------------------------------
Gradle ${version.version}
------------------------------------------------------------

Build time:   $version.buildTimestamp
...
"""
```

Reply:
Seems like it could.  This is an existing test, IDEA just adjusted the spacing.  If we end up merging this branch I'll change these expected output blocks to multiline strings here first.